source/2020-09-01-ember-3-21-released.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Ember 3.21 Released
+author: Kenneth Larsen, Isaac Lee
+tags: Releases, 2020, 3, 3.21, Version 3.x
+responsive: true
+---
+
+Today the Ember project is releasing version 3.21 of Ember.js, Ember Data, and Ember CLI.
+
+Version 3.20 of Ember is now promoted to LTS (Long Term Support). An LTS version of Ember continues to receive security updates for 9 release cycles (54 weeks) and bugfixes for 6 cycles (36 weeks). LTS releases typically occur every four minor versions. The previous LTS version of Ember was 3.16.
+
+This release kicks off the 3.22 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/releases/)
+- [The Ember Release Cycle](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html)
+- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.21
+
+Ember.js 3.21 is an incremental, backwards compatible release of Ember with bugfixes, performance improvements, and minor deprecations.
+
+#### New Features
```

Comment:
Not a blocker at all, but it looks to me like we have typically added the number of features/deprecations in parens in the heading, even if it's zero. Maybe we decided to move away from this though?
```suggestion
#### New Features (0)
```

Reply:
Yeah, I removed the numbers from the headings.

Reply:
FWIW, historically there has been a lot of controversy over the numbering in #core-meta (sometimes the # count was wrong, or the # didn't seem that helpful), seems good to get rid of it 👍